azure-pipelines.yml

```diff
@@ -256,6 +263,10 @@ jobs:
         - Windows_NT_x64
         - Windows_NT_x86
         helixQueueGroup: all
+      ${{ if eq(variables['Build.DefinitionName'], 'coreclr-gc-reliability-chk') }}:
+        platforms:
+        - Windows_NT_x64
+        helixQueueGroup: all
```

Comment:
I would avoid using `all` for `helixQueueGroup` this could block all the queues for hours from running - just use `ci`.